include_core/ute_dataformat.h

```diff
@@ -145,6 +146,7 @@ typedef enum {
 	UT_IA64,
 	UT_S390X,
 	UT_AMD64,
+	UT_RISC,
```

Comment:
Similar here, the architecture is RISC-V so this should be something like `UT_RISCV` (rather than `UT_RISC`, i.e., without `V` at the end)

Reply:
Agreed and updated to `UT_RISCV` for the architecture as suggested above.